SuperPong/Assets/Scripts/BallController.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BallController : MonoBehaviour
+{
+    private Rigidbody sphere;
+    private int velocityRange = 200;
+    private float initialHeight;
+
+	// Use this for initialization
+	void Start ()
+    {
+        sphere = GetComponent<Rigidbody>();
+        initialHeight = sphere.position.y;
+        resetBall(sphere);
+        sphere.AddForce(new Vector3(Random.Range(velocityRange, velocityRange*2), 0, Random.Range(velocityRange, velocityRange*2)));
```

Comment:
I downloaded the branch and played around with it just a little bit, if the ball levitates off the ground and its y position is constrained, then we can use raw velocities and not worry about forces since the ground won't apply any friction.